src/main/kotlin/no/item/gradle/InitRunnerExtension.kt

```diff
@@ -0,0 +1,54 @@
+package no.item.gradle
+
+
+import org.gradle.api.Action
+import org.gradle.api.model.ObjectFactory
+import javax.inject.Inject
+
+@Suppress("UnstableApiUsage") // ObjectFactory is incubating
+open class InitRunnerExtension @Inject constructor(
+    objectFactory: ObjectFactory
+) {
+
+    // region Model
+
+    var coverageTypes = listOf("instruction")
```

Comment:
Just remove this, if it's not intended to be used for what we are doing.

Reply:
Configs we might be interested in taking in, includes a black list over xml-files we don't want to generate code from.

Reply:
Removed a bunch of unnecessary code here. For now the class is moslty empty, but keeping it for possibilty of configs later.